core/src/main/java/org/mindtrails/persistence/ActionLogRepository.java

```diff
@@ -0,0 +1,30 @@
+package org.mindtrails.persistence;
+
+import org.mindtrails.domain.tracking.ActionLog.ActionLog;
+import org.mindtrails.domain.Participant;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Date;
+import java.util.List;
+
+@Repository
+public interface ActionLogRepository extends JpaRepository<ActionLog, Long> {
+    Long countByDateRecordedAfter(Date date);
+
+    public List findByDateRecordedGreaterThan(Date date);
+
+//    Allows us to calculate avg latency of a particular task, across all participants
```

Comment:
Don't stress about this, it just caught my eye.  I was thinking we had duration stored in the model, not just time stamps.  Aggregate functions wouldn't be helpful here given the current data structure.

Reply:
Ah - I think I was overly confusing with these comments...
 
We just have time stamps at the moment, because I remember you had concerns about calculating the latency on the Java side of things. So I guess my brain thought it was best to only retain the time at which an action was recorded, with the intent of calculating the latency, average latency, etc later on in python (which Sonia seemed to like). 

But wow! in this case, yeah...it looks like maybe I should take your advice and send the latency from the client side, store it in the database, and spit out the average using the aggregate functions. Let me get this in place and get back to you.